lib/rspec/support/ruby_features.rb

```diff
@@ -65,6 +65,25 @@ def supports_exception_cause?
         end
       end
 
+      if Ruby.mri? && RUBY_VERSION >= '1.9.0'
+        def ripper_supported?
+          true
+        end
+      elsif Ruby.jruby? &&
+            defined?(Gem::Version) &&
+            Gem::Version.new(JRUBY_VERSION) >= Gem::Version.new('1.7.14') &&
+            !JRUBY_VERSION.start_with?('9.')
+        # JRuby 1.7.5 - 1.7.13 actually supports Ripper but it's too slow.
```

Comment:
From my understanding Ripper on JRuby performs parsing purely in runtime so I don't think it's a compile time issue.